CONTRIBUTING.md

```diff
@@ -1,3 +1,68 @@
-If you submit a pull request, please adhere to Contributor Guidelines:
+## Contributor's Guidelines & Code Conventions
 
-https://github.com/micropython/micropython-lib/wiki/ContributorGuidelines
+micropython-lib follows the same general conventions as the [main MicroPython
+repository](https://github.com/micropython/micropython). Please see
+[micropython/CONTRIBUTING.md](https://github.com/micropython/micropython/blob/master/CONTRIBUTING.md)
+and [micropython/CODECONVENTIONS.md](https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md).
+
+### Raising issues
+
+Please include enough information for someone to reproduce the issue you are
+describing. This will typically include:
+
+* The version of MicroPython you are using (e.g. the firmware filename, git
+  hash, or version info printed by the startup message).
+* What board/device you are running MicroPython on.
+* Which package you have installed, how you installed it, and what version.
+  When installed via `mip`, all packages will have a `__version__`
+  attribute.
+* A simple code snippet that demonstrates the issue.
+
+If you have a how-to question or are looking for help with using MicroPython
+or packages from micropython-lib, please post at the[discussion forum]
+(https://github.com/orgs/micropython/discussions) instead.
+
+### Pull requests
+
+The same rules for commit messages, signing-off commits, and commit structure
+apply as for the main MicroPython repository. All Python code is formatted
+using `black`. See `tools/codeformat.py` to apply `black` automatically
```

Comment:
```suggestion
using `black`. See [`tools/codeformat.py`](https://github.com/micropython/micropython-lib/blob/top-level-docs/tools/codeformat.py) to apply `black` automatically
```

Reply:
Same. But I think it makes more sense as a link than as `code` so will fix both.